### DISCUSSION TITLE
iOS15 - Speed up your typing (overlay)

### DISCUSSION THREAD

**Discussion by RSM-SBA** [appium/appium#16177] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/90029825/143880763-607c2c63-7cb8-49c9-a493-fc857aa3c954.png)

Has anyone an automated way to get rid of this help overlay, which appears when clicking into an input field  ?

**Comment by KazuCocoa** [comment, marked as answer]
Is there a way to show it? So far, i haven't seen such tutorial on iOS, so would like to know when it appears.
e.g. when create a new simulator from scratch with Xcode XX and iOS 15.2 etc

Appium tries to disable some tutorial cases. If it is general preference in iOS and it can be disabled via like the blow key, Appium/WDA can have it.
https://github.com/appium/WebDriverAgent/blob/a1824895274c60798b506f31054e4ce1e0400c71/WebDriverAgentLib/Utilities/FBConfiguration.m#L263-L270

Generally, you should click the "Continue" button.

**Reply by RSM-SBA** [reply]
it appears in a fresh 15.0 Simulator. 
It appears only once when you click into an input field (maybe when the keyboard is shown the first time)
It will only reappear if you recreate a new simulator (or erase all settings and content fro the device).

As i create a new simulator for each testrun, i always see this dialog on the first test, and as i never now which test runs first, i have make a check for this dialog on every input field interaction...

@KazuCocoa what do you mean with blow key ? Is that a capability ?

**Reply by KazuCocoa** [reply]
https://github.com/appium/appium-ios-simulator/pull/315 will disable it.

> what do you mean with blow key ? Is that a capability ?

By default, Appium/WDA set some keys to disable tutorials to avoid this issue, but It looks like iOS 15+ changed the way to apply. The simulator 315 PR will do the same thing for iOS 15+.

**Reply by RSM-SBA** [reply]
ah ok, thank you :)

**Reply by KazuCocoa** [reply]
`appium-ios-simulator@3.29.0` has the change.
You can use it via `appium@beta` for now.

**Reply by RSM-SBA** [reply]
Thank you. My testsuites are running on the appium 2.0 beta so i think i have to wait for the next beta release. correct ?

**Reply by KazuCocoa** [reply]
Oh, great. Then, probably you can get the version after uninstalling current xcuitest-driver and install it again

**Reply by RSM-SBA** [reply]
i give it a try :)

**Reply by RSM-SBA** [reply]
hmm but it is still  3.58 or do i have to use @beta for the driver installation
✔ Installing 'xcuitest' using NPM install spec 'appium-xcuitest-driver'
Driver xcuitest@3.58.0 successfully installed

**Reply by RSM-SBA** [reply]
seems to work now ! great ! 
Big thanks for the quick fix/responses

**Reply by KazuCocoa** [reply]
yes. the dependency was allowed to up-to-date every time.
